static/src/javascripts/projects/commercial/modules/dfp/lazy-load.js

```diff
@@ -21,7 +21,11 @@ define([
     /* loadQueued: boolean. Set to true when a lazyload task is scheduled */
     var loadQueued = false;
 
-    var lazyLoad = throttle(function () {
+    var lazyLoad = dfpEnv.lazyLoadObserve ? onIntersect : throttle(onScroll, nbOfFrames * durationOfFrame);
```

Comment:
yes, probably. I wanted to do something along the lines of what the polyfill does with `THROTTLE_TIMEOUT` but honestly I don't know if it makes any difference. Will switch to `throttledScroll`.